Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocTextValue.css

```diff
@@ -45,48 +45,28 @@ span.bocTextValue .content
   display: inline-block;
 }
 
-.bocTextValue.readOnly .content span[tabindex]:first-child
-{
-  /* Ensures that diacritic marks can be rendered without being cut off. */
-  padding-top: 0.2em;
-}
-
-.bocTextValue.readOnly .content span[tabindex]:last-child
-{
-  /* Ensures that diacritic marks can be rendered without being cut off. */
-  padding-bottom: 0.2em;
-}
-
 .bocTextValue.readOnly .content span[tabindex]:empty
 {
-  /* Ensures that empty readonly values are not collapsed and can therefor visualize the keyboard focus */
-  width: calc(100% - 2px);
+  /* Ensures that empty readonly values are not collapsed and can therefore visualize the keyboard focus */
+  width: 100%;
   min-height: 1.5em;
-  padding-top: 0;
-  padding-bottom: 0;
-  vertical-align: -0.2em;
+  vertical-align: middle;
 }
 
 span.bocTextValue input
 {
   width: 100%;
+  min-height: inherit;
+  height: 1.2em;
+  line-height: 1em;
 }
 
 span.bocTextValue textarea
 {
   width: 100%;
-  height: 100%;
+  /* Forcing assignment of height from parent and overriding height set in code-behind. */
+  height: 100% !important;
```

Comment:
Please link the place in the code-behind that's responsible for this. if the code-behind determines the height based on the number of rows, then the code-behind is correct. If it's just some fixed value, we should move it to css in both themes.

Reply:
If I am not mistaken, then it should be this line (and the one above): [https://github.com/re-motion/Framework/blob/develop/Remotion/ObjectBinding/Web.Test/IndividualControlTests/BocTextValueUserControl.ascx#L116](url). Unless the code is auto-generated, the values for height (6em and 5em) are hard-coded and could be moved to css, I assume.

Reply:
This 5em are user-defined and should not be overruled.